src/Apps/Components/Layouts/LayoutLogoOnly.tsx

```diff
@@ -15,17 +14,15 @@ export const LayoutLogoOnly: FC<BaseLayoutProps> = ({ children }) => {
 
       <LayoutMain>
         <AppContainer>
-          <HorizontalPadding>
```

Comment:
Removed this bc `LayoutMain` also has horizontal padding

Reply:
It does? https://github.com/artsy/force/blob/main/src/Apps/Components/Layouts/Components/LayoutMain.tsx

Reply:
oh.. its `LayoutContainterOnly` which I see is not inside the `LogoOnly` 🤔

Reply:
This looks well aligned with the other layouts. Something else is likely causing the double padding, not the layout. I'd roll back this change. https://force-storybook.artsy.net/?path=/story/components-layouts--logo-only

Reply:
Oh I see whats happening it's due to the routes being `ContainerOnly`.

Reply:
Ah, yep. Generally speaking we'd want to apply the layout using the routes. Actually it hadn't occurred to me that anyone would want to use them otherwise. So it seems like you're going to have two `main` elements, with duplicated IDs, which you don't want.

Reply:
(And duplicate toast containers)

If you wanted to use these outside of routing I suppose one thing you could do is maybe add a `LayoutNone` option (or whatever) that is literally just `children`.

Reply:
Good idea i've added back the padding to `LogoOnly` and created a `LayoutNone` for the order routes